<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="de.bertelsmann.client.rest.kasse20.tc516.J516_OnlineNachkreditierungMitRetoure" tests="4" skipped="0" failures="0" errors="0" timestamp="2025-03-04T17:17:35" hostname="EISMANN" time="1.696">
  <properties/>
  <testcase name="testError" classname="de.bertelsmann.client.rest.kasse20.tc516.J516_OnlineNachkreditierungMitRetoure" time="1.4">
    <system-out><![CDATA[initializing... (initContext)class de.bertelsmann.client.rest.kasse20.tc516.J516_OnlineNachkreditierungMitRetoure
initializing... (SetupOnce)class de.bertelsmann.client.rest.kasse20.tc516.J516_OnlineNachkreditierungMitRetoure
initializing... (initContext)class de.bertelsmann.client.rest.kasse20.tc516.J516_OnlineNachkreditierungMitRetoure
]]></system-out>
    <system-err><![CDATA[MÃ¤r 04, 2025 6:17:37 PM org.glassfish.jersey.test.inmemory.InMemoryTestContainerFactory$InMemoryTestContainer <init>
INFORMATION: Creating InMemoryTestContainer configured at the base URI http://localhost:9998/
]]></system-err>
  </testcase>
  <testcase name="testInvalidRequest" classname="de.bertelsmann.client.rest.kasse20.tc516.J516_OnlineNachkreditierungMitRetoure" time="0.232"/>
  <testcase name="testInvalidToken" classname="de.bertelsmann.client.rest.kasse20.tc516.J516_OnlineNachkreditierungMitRetoure" time="0.031">
    <system-out><![CDATA[ERROR #1741108657356
ERROR #1741108657357
ERROR #1741108657357
ERROR #1741108657358
]]></system-out>
  </testcase>
  <testcase name="testSuccess" classname="de.bertelsmann.client.rest.kasse20.tc516.J516_OnlineNachkreditierungMitRetoure" time="0.033"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
